src/opera_disp_tms/generate_metadata_tile.py

```diff
@@ -65,14 +65,14 @@ def buffer_frame_geometry(frame: Frame, buffer_size_in_meters: int = -3500) -> F
     return frame
 
 
-def reorder_frames(frame_list: Iterable[Frame], order_by: str = 'west_most') -> List[Frame]:
+def reorder_frames(frame_list: Iterable[Frame], add_first: str) -> List[Frame]:
     """Reorder a set of frames so that they overlap correctly when rasterized.
     Frames within a relative orbit are stacked so that higher frame numbers are on top (so they are rasterized first).
-    Relative orbits sets are orderd by either frame number, or from west to east.
+    Relative orbits sets are orderd by either frame number, from west to east, or from east to west.
 
     Args:
         frame_list: The list of frames to reorder
-        order_by: Strategy to use when ordering relative orbit sets (`frame_number`, or `west_most`)
+        add_first: Strategy to use when ordering relative orbit sets ("min_frame_number", "east_most", "west_most")
```

Comment:
When would min_frame_number be used? Would setting a default make sense at all?

Reply:
It would largely be used for testing, but not production. I could see removing it in the future.